content/guides/author-libraries.md

```diff
@@ -6,14 +6,57 @@ contributors:
   - johnstew
   - simon04
   - 5angel
+  - marioacc
 ---
 
-webpack is a tool which can be used to bundle application code and also to bundle library code. If you are the author of a JavaScript library and are looking to streamline your bundle strategy then this document will help you.
+webpack is a tool which can be used to bundle application code and also to bundle library code. If you are the author of a JavaScript library and are looking to streamline your bundle strategy then this document will help you with the differents webpack configurations to expose your libraries as you think convenient.
```

Comment:
s/differents/different

g/as you think/as you think is

Reply:
Perfect, will make the changes.

Reply:
@marioacc I would actually probably simplify/rephrase this as:

> Aside from applications, webpack can also be used to bundle JavaScript libraries. The following guide is meant for library authors looking to streamline their bundling strategy.

What do you think?

Reply:
I like it. The simpler, the better.